pyo3-derive-backend/src/pymethod.rs

```diff
@@ -193,7 +193,7 @@ pub fn impl_wrap_new(cls: &syn::Type, spec: &FnSpec<'_>) -> TokenStream {
     quote! {
         #[allow(unused_mut)]
         unsafe extern "C" fn __wrap(
-            _cls: *mut pyo3::ffi::PyTypeObject,
+            subcls: *mut pyo3::ffi::PyTypeObject,
```

Comment:
I think it's better to keep the `cls` name convention maybe.

```suggestion
            cls: *mut pyo3::ffi::PyTypeObject,
```

Reply:
![image](https://user-images.githubusercontent.com/16046705/85241186-f1418e00-b475-11ea-9acc-8cdb6d18fb52.png)
Maybe `subtype` is better?

Reply:
Ah yes good find 👍 I think subtype is probably better than `cls` here and also for `create_cell_with_type` in that case!